src/js/lib/content/src/index.js

```diff
@@ -179,20 +179,37 @@ const MAX_MESSAGE_LENGTH = 500;
 
 function remoteLog(event,data) {
     if (mixpanel) {
-        event = JSON.stringify(event).substr(0,MAX_MESSAGE_LENGTH)
-        if (data !== undefined) {
+        event = event.substr(0,MAX_MESSAGE_LENGTH)
+        if (data !== undefined && data !== null) {
```

Comment:
function to check whether soething is undefined or null should be extracted and used everywhere. In the code below we chach `(data !== undefined)` - we should also check if it was null. Basically, checking for both is a good practice always.

Reply:
This is now only in one method, which is re-used to do all the logging.